src/assets/audio/player.js

```diff
@@ -0,0 +1,17 @@
+let audio = new Audio('');
```

Comment:
Dlaczego to jest w assetach? Moim i Aleksego zdaniem Player powinien komponentem lub częścią większego komponentu któremu przekazuje się ścieżke dźwięku. Jeśli te funkcje używasz w kilku komponentach to najlepiej przenieść je do folderu np 'utils, helpers'. Jeśli używasz to w jednym koponencie to umieść to tam. 

Jeśli przeniesiesz do utilsów funkcje powinny być definiowane

```
const playAudio = (src) = {
}

export default {
   playAudio,
   pauseAudio
}